tools/inspect/apple_macro_check.cpp

```diff
@@ -90,8 +92,6 @@ namespace boost
                   line_start = it + 1; // could be end()
               }
           }
-
-          std::string lineloc = linelink(full_path, std::to_string(line_number));
```

Comment:
It works for both apple_macro and ascii check, as both have four arguments, the last one having three.  If the link works, https://github.com/Bcorde5/hpx/blob/HyperlinkFixes/tools/inspect/inspect.cpp#L547-L555 is where the overload occurs.  This only works, and the only reason it must work is because in both of these checkers, the line number is not included in the string, but rather as a size_t argument.  The other checkers cannot use a similar system because they either do not have a line number or the line number is within the string, which makes it easier to point out multiple mistakes in a single run on inspect, but makes it tricky creating such an overload for the other checkers with only three arguments.